Updated failing tests from develop

AndreiBrehuescu opened the pull request:
There were some tests taht keep failing when maven check was runed using github actions.
Most of them were caused by the order they were executed. I refactored the code so that the tests don't require a specific order for them to succeed.

brehua committed:
Updated failing tests from develop

ZbRoby commented on event-management-backend/src/test/java/ro/msg/event_management/SaveEventIntegrationTest.java:
```diff
@@ -65,27 +81,21 @@ void testSaveEvent() {
-                .endDate(LocalDate.now())
-                .startHour(LocalTime.now())
-                .endHour(LocalTime.now().plusHours(3))
+                .startDate(LocalDate.parse("2020-11-11"))
+                .endDate(LocalDate.parse("2020-11-15"))
+                .startHour(LocalTime.parse("18:00"))
+                .endHour(LocalTime.parse("20:00"))
```
I do not see a reason to change to specific dates.

AndreiBrehuescu replied:
Switched back to dates relative to the current time.

ZbRoby commented on event-management-backend/src/test/java/ro/msg/event_management/SaveEventIntegrationTest.java:
```diff
@@ -65,27 +81,21 @@ void testSaveEvent() {
                 .creator("")
                 .picturesUrlSave(picturesUrlSave)
                 .ticketCategoryDtoList(ticketCategoryDtoList)
                 .location(1)
```
What would this '.location(1)' point to?
<details>
  <summary>Spoiler warning</summary>
This location does not seem to be used in the tested logic. Either remove it or point to the created location.
</details>

AndreiBrehuescu replied:
It was useless because the service get the location Id for insert. I removed it.

ZbRoby commented on event-management-backend/src/test/java/ro/msg/event_management/AdminHomepageIntegrationTests.java:
```diff
@@ -56,6 +57,17 @@
+    }
+
     @Test
     public void test_card_with_events() {
```
From your side is a good temporary fix. The issue if with this test is that is not that descriptive. Right now it technically only test if there is an event and if that event is before or equal with himself.

(I only wanted to highlight the method name)

AndreiBrehuescu replied:
Changed the method name to a more descriptive one.

ZbRoby reviewed (commented):
From my side the fix is ok. I found some problems with the tests themselves, but is not part of the given task. Maybe look on the points I made, but do not take to much time on them since it is not the priority in this sprint.

brehua committed:
Update SaveEventIntegrationTest.java

Changed the name of the method to be more descripitve.

brehua committed:
Revert "Update SaveEventIntegrationTest.java"

This reverts commit cd03aa0a3ca5461ea04d6fd5dc62e35676f80b5a.

brehua committed:
Revert "Revert "Update SaveEventIntegrationTest.java""

This reverts commit 105de3426eabc9bf6ae91959e30de9a493aac942.

brehua committed:
Update AdminHomepageIntegrationTests.java

Updated the name of the method to be more descritive.

brehua committed:
Update SaveEventIntegrationTest.java

Removed location set because was  not used. And switched back to relative dateTIme

brehua committed:
Update AdminHomepageIntegrationTests.java

Changed the name of the method to be explicit and changed the way assert is handled
